src/ui/components/SuggestedResults/style.scss

```diff
@@ -0,0 +1,34 @@
+@import "~ui/css/vars";
+$item-padding: 5px 52px;
+
+.SuggestedResults {
+  background-color: $white;
+  color: $link-color;
+  padding-bottom: 10px;
+  padding-top: 10px;
+
+  &-list {
```

Comment:
Huh. I didn't know about this but it makes sense.

That said I think explicitly writing `.SuggestedResults-item { [...styles] }` is better: it's easier to grep for and easier to immediately understand.